src/main/java/io/cdap/e2e/pages/actions/CdfNameSpaceAdminActions.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.actions;
+
+import io.cdap.e2e.pages.locators.CdfNameSpaceAdminLocators;
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.JsonUtils;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Map;
+
+/**
+ * Represents CdfNameSpaceAdminActions
+ */
+public class CdfNameSpaceAdminActions {
+
+  private static final Logger logger = LoggerFactory.getLogger(CdfNameSpaceAdminActions.class);
+  public static CdfNameSpaceAdminLocators cdfNameSpaceAdminLocators;
+
+  static {
+    cdfNameSpaceAdminLocators = SeleniumHelper.getPropertiesLocators(
+        CdfNameSpaceAdminLocators.class);
+  }
+
+  public static void clickOnNameSpaceAdminTabs(String tabName, String nameSpaceName) {
+    ElementHelper.clickOnElement(
+        CdfNameSpaceAdminLocators.nameSpaceModules(tabName, nameSpaceName));
+  }
+
+  public static void clickOnNameSpaceAdminTab() {
```

Comment:
Change the method as this and the above one has the same naming only having added s letter.

Reply:
yes changed the method name to openNamespaceAdminPage()